### ISSUE TITLE
A11y_Azuresettings_ScreenReader: Voice over does not announce the label for the" Loctaion,VM count,VM size and aks cluster name"edit box . 

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7937]
GitHubTags: #SQL Azure Data Studio;#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1,#WCAG4.1.2;#WCAG3.3.2;#DesktopApp;#Voice Over;#Mac
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS:  MacOS Mojave 10.14.6
**Additional Details:**
Voice Over 

**Steps to Reproduce :**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button.

7.Open Voce over and observe the screen reading.
**Actual:**
Voice over does not announce the label for the " Loctaion,VM count,VM size and aks cluster name" edit box .. 
**Expected:**
Voice over should announce the label for the " Loctaion,VM count,VM size and aks cluster name" edit box .. 
**User Impact:**
Screen reader User will not able to relate with the edit  and combo box screen reader user will not able to understand what they need to fill in the edit box.
**Attachment for reference**
[7937_A11y_AzureDataStudio_AzureSettings_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3760686/7937_A11y_AzureDataStudio_AzureSettings_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
I think this is a VoiceOver issue, we already have aria-label set for all the input controls, actually if you use CAPS left or right keys and go through the controls, sometimes it will read the aria-label sometimes not.
